examples/01-produce-key-value/src/main.rs

```diff
@@ -0,0 +1,40 @@
+//! A minimal example showing how to produce key-value records with Fluvio
+//!
+//! Before running this example, make sure you have created a topic
+//! named `key-value` with the following command:
+//!
+//! ```text
+//! $ fluvio topic create key-value
+//! ```
+//!
+//! Run this example using the following:
+//!
+//! ```text
+//! $ cargo run --bin produce-key-value
+//! ```
+//!
+//! After running this example, you can see the messages that have
+//! been sent to the topic using the following command:
+//!
+//! ```text
+//! $ fluvio consume key-value -B -d
+//! [Hello] Fluvio
+//! ```
+
+#[async_std::main]
```

Comment:
Can we collapse this into produce example? (with different binary). Otherwise, we are starting to create too many example crates.
Or alternatively, combine with original produce example

Reply:
The main reason I did not combine the two is because the regular `produce` example is actually used in a test. So there wasn't really a great way to keep the examples small and clean while also satisfying the test and putting both together.

Reply:
fair enough.